docs/tutorials/dynamics_backend.rst

```diff
@@ -0,0 +1,566 @@
+
+    %%time
+    rabi0_data = rabi0.run().block_for_results()
+    rabi1_data = rabi1.run().block_for_results()
+
+Plot the results.
+
+.. jupyter-execute::
+
+    rabi0_data.figure(0)
+
+.. jupyter-execute::
+
+    rabi1_data.figure(0)
+
+Observe the updated parameters for qubit 0.
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[0, ()], parameters="amp"))
+
+5.5 Rough Drag parameter calibration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run rough Drag parameter calibration for the ``X`` and ``SX`` gates. This follows the same procedure
+as above.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library.calibration import RoughDragCal
+    
+    cal_drag0 = RoughDragCal(0, cals, backend=backend, betas=np.linspace(-20, 20, 15))
+    cal_drag1 = RoughDragCal(1, cals, backend=backend, betas=np.linspace(-20, 20, 15))
+    
+    cal_drag0.set_experiment_options(reps=[3, 5, 7])
+    cal_drag1.set_experiment_options(reps=[3, 5, 7])
+    
+    cal_drag0.circuits()[5].draw(output="mpl")
+
+.. jupyter-execute::
+
+    %%time
+    drag0_data = cal_drag0.run().block_for_results()
+    drag1_data = cal_drag1.run().block_for_results()
+
+.. jupyter-execute::
+
+    drag0_data.figure(0)
+
+
+.. jupyter-execute::
+
+    drag1_data.figure(0)
+
+
+5.6 Fine amplitude calibration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Finally, run fine amplitude calibration for both qubits. Start with the ``X`` gate.
+
+.. jupyter-execute::
+
+    from qiskit_experiments.library.calibration.fine_amplitude import FineXAmplitudeCal
+    
+    amp_x_cal0 = FineXAmplitudeCal(0, cals, backend=backend, schedule_name="x")
+    amp_x_cal1 = FineXAmplitudeCal(1, cals, backend=backend, schedule_name="x")
+    
+    amp_x_cal0.circuits()[5].draw(output="mpl")
+
+
+.. jupyter-execute::
+
+    %%time
+    data_fine0 = amp_x_cal0.run().block_for_results()
+    data_fine1 = amp_x_cal1.run().block_for_results()
+
+.. jupyter-execute::
+
+    data_fine0.figure(0)
+
+
+.. jupyter-execute::
+
+    data_fine1.figure(0)
+
+Next, run fine calibration on the ``SX`` gates.
+
+.. jupyter-execute::
+
+    # Do SX Cal
+    from qiskit_experiments.library.calibration.fine_amplitude import FineSXAmplitudeCal
+    
+    amp_sx_cal0 = FineSXAmplitudeCal(0, cals, backend=backend, schedule_name="sx")
+    amp_sx_cal1 = FineSXAmplitudeCal(1, cals, backend=backend, schedule_name="sx")
+    
+    amp_sx_cal0.circuits()[5].draw(output="mpl")
+
+
+.. jupyter-execute::
+
+    %%time
+    data_fine_sx0 = amp_sx_cal0.run().block_for_results()
+    data_fine_sx1 = amp_sx_cal1.run().block_for_results()
+
+.. jupyter-execute::
+
+    pd.DataFrame(**cals.parameters_table(qubit_list=[0, ()], parameters="amp"))
+
+6. Simulating 2 qubit interaction characterization via the ``CrossResonanceHamiltonian`` experiment
+---------------------------------------------------------------------------------------------------
+
+Finally, simulate the ``CrossResonanceHamiltonian`` characterization experiment.
```

Comment:
You can link directly to the experiments API docs for this. Also, maybe add a brief explanation for why the backend needs more configuration to run this experiment.

Reply:
Just added `qiskit_experiments` to intersphinx (hopefully correctly) and turned `CrossResonanceHamiltonian` into a link. I've also expanded the bullet points below to give more context as to why the experiment requires this further configuration:
- Adding the gate to the target is required for validation
- The control_channel_map is required for the experiment to know which control channel to drive for a given control-target cross resonance pair